### ISSUE TITLE
add `charts` filter to the health template

### ISSUE THREAD

**Issue by ilyam8** [netdata/netdata#11000]
##### Feature idea summary

> Templates define rules that apply to all charts of a specific context, and use the template label. Templates help you apply one entity to all disks, all network interfaces, all MySQL databases, and so on.

> apply to all charts of a specific context

The following [alarm entities](https://learn.netdata.cloud/docs/agent/health/reference#entity-format) allow to restrict a template a certain chart:
 - [os](https://learn.netdata.cloud/docs/agent/health/reference#alarm-line-os)
 - [hosts](https://learn.netdata.cloud/docs/agent/health/reference#alarm-line-hosts)
 - [plugin](https://learn.netdata.cloud/docs/agent/health/reference#alarm-line-plugin)
 - [module](https://learn.netdata.cloud/docs/agent/health/reference#alarm-line-module)
 - [families](https://learn.netdata.cloud/docs/agent/health/reference#alarm-line-families)
 - [host labels](https://learn.netdata.cloud/docs/agent/health/reference#alarm-line-host-labels)

There is nothing that allows restricting a template to a specific chart id or charts ids.

Use cases: 
 - [several jobs of the same collector and you want to apply same alarm, but with different trigger conditions](https://community.netdata.cloud/t/different-alarm-settings-for-different-web-log-conf/1174).
   - jobs charts ids contain the job name
 - you want to exclude some pods/containers from health monitoring
   - cgroups charts ids contain pod/container name

Adding a filter that restricts a template to a specific chart solves the problem.

##### Expected behavior
 - there is `charts` health template entity
 - its sytnax is [simple pattern](https://learn.netdata.cloud/docs/agent/libnetdata/simple_pattern)

Basically same as [exporting engine](https://learn.netdata.cloud/docs/agent/exporting) `send charts matching`

---

@manos-saratsis that is the idea. I am assigning you, so you can think about it and approve/change/cancel.

@thiagoftsm for my understanding it is relatively easy/fast to implement, am I correct?

**Comment by stelfrag** [comment]
Ping @MrZammler

**Comment by thiagoftsm** [comment]
@ilyam8  you are right, we already worked with something like this in PR https://github.com/netdata/netdata/pull/10041 .
